extensions/objects/templates/FilepathEditorDirective.ts

```diff
@@ -17,6 +17,7 @@
  */
 
 // This directive can only be used in the context of an exploration.
+require('components/forms/custom-forms-directives/image-uploader.directive.ts');
```

Comment:
Where is this used?

Reply:
It is used at `extensions/objects/templates/filepath_editor_directive.html`